requirements.txt

```diff
@@ -2,3 +2,5 @@ beautifulsoup4==4.11.1
 icalendar==4.0.9
 icalevents==0.1.25
 recurring_ical_events==1.0.2b0
+requests==2.28.1
```

Comment:
Sorry - I'm not the export for requirements.txt. Could it cause trouble with other integrations if we tag frequently used Python modules like `requests` with a fixed number?